Pages/userInformation.js

```diff
@@ -0,0 +1,5 @@
+export const myPersonalDetails = {
```

Comment:
this file is not a page object model, so you should place it somewhere else, not in the Pages folder. Maybe create a different folder named data or constants and move it there.